module/VuFind/src/VuFind/Db/Entity/Search.php

```diff
@@ -315,6 +355,7 @@ public function getSearchObject(): ?\VuFind\Search\Minified
     public function setSearchObject(?\VuFind\Search\Minified $searchObject): static
     {
         $this->searchObject = $searchObject ? serialize($searchObject) : null;
+        $this->deserializedSearchObject = null;  
```

Comment:
Should we store the unserialized object here?

```suggestion
        $this->deserializedSearchObject = $searchObject;  
```